config/manager/deployment.yaml

```diff
@@ -20,6 +20,10 @@ spec:
         prometheus.io/port: "8080"
     spec:
       terminationGracePeriodSeconds: 10
+      # Required for AWS IAM Role bindings
+      # https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-technical-overview.html
+      securityContext:
+        fsGroup: 1337
```

Comment:
That should be: 65533

Reply:
```
/ $ id controller
uid=100(controller) gid=65533(nogroup) groups=65533(nogroup),65533(nogroup)
```

Reply:
nop, fsGroup is irrelevant

Reply:
ah, okay strange.

Reply:
Probably going to need the same fix for the other controllers that do not have the fsGroup set and access AWS resources.

Reply:
`notification-controller` and `helm-controller` cannot access the token in my setup for example.

Reply:
Yes but each of those should run under their own service account, are you using an older version of flux or have you bind their SAs too?

Reply:
Would you mind telling me why do you need to bind the `notification-controller` and  `helm-controller` service account to an IAM Role? None of these controllers should make API calls to AWS.

Reply:
> Yes but each of those should run under their own service account, are you using an older version of flux or have you bind their SAs too?

You are right we are running an older version of flux. We have been provisioning our cluster using the `flux` terraform provider. I believe the latest version. I will update and see how that goes.

Reply:
> Would you mind telling me why do you need to bind the `notification-controller` and `helm-controller` service account to an IAM Role? None of these controllers should make API calls to AWS.

If that's the case then we shouldn't need it, although if any of those controllers are extended then it would be ideal if they supported IRSA out of the box. 

Thank you for the quick fix! We are relying heavily on this project for our success.